example.yml

```diff
@@ -0,0 +1,188 @@
+---
+region: eu-west-1
+metrics:
+- aws_namespace: AWS/ELB
+  aws_metric_name: HealthyHostCount
+  aws_dimensions:
```

Comment:
Hmm, I think the length of this file might be a little scary to users, would you mind doing `aws_dimensions: [AvailabilityZone, LoadBalancerName]` and the same for statistics?